esmvaltool/preprocessor/_mask.py

```diff
@@ -1,16 +1,20 @@
 """
-_mask.py
+Mask module.
 
-module that performs missing values masking
-and geographical area eslection
+Module that performs a number of masking
+operations that include: masking with fx files, masking with
+Natural Earth shapefiles (land or ocean), masking on thresholds,
+missing values masking.
 """
 
 import logging
 import os
 
+import dask.array as da
+import numpy as np
```

Comment:
imports are not correctly sorted

Reply:
I used `isort` for this, and that's what it did, what is the correct order?

Reply:
https://www.python.org/dev/peps/pep-0008/#imports